com.unity.visualeffectgraph/Editor/Models/Parameters/VFXDynamicBuiltInParameter.cs

```diff
@@ -179,6 +179,19 @@ override public string name
             }
         }
 
+        public override VFXCoordinateSpace GetOutputSpaceFromSlot(VFXSlot outputSlot)
+        {
+            switch (m_BuiltInParameters)
+            {
+                case BuiltInFlag.LocalToWorld:
+                    return VFXCoordinateSpace.Local;
+                case BuiltInFlag.WorldToLocal:
+                    return VFXCoordinateSpace.World;
+                default:
+                    return (VFXCoordinateSpace)int.MaxValue;
```

Comment:
Optionally, if `outputSlot.spaceable` and you reach the default state, you could throw an invalid operation exception.